bin/llvm-krun

```diff
@@ -472,7 +484,7 @@ set +e
 if [ "$verbose" -ge 1 ]; then
   set -x
 fi
-$debug "$dir"/interpreter "$expanded_input_file" "$depth" "$output_file" "${interpreter_flags[@]}"
+debug+=("${dir}/interpreter" "$expanded_input_file" "$depth" "$output_file" "${interpreter_flags[@]}")
```

Comment:
I don't understand this line. Previously, it was _invoking_ the debugger but here it's appending to an array. Where is the debugger actually executed here?

Reply:
Good point! Sorry for the mistake, I've just updated it!